packages/js/product-editor/src/blocks/product-fields/downloads/edit-downloads-modal/style.scss

```diff
@@ -15,7 +15,26 @@
 		padding-bottom: $gap-large;
 	}
 
+	&__preview {
+		margin-bottom: $gap;
+	}
+
 	.components-input-control__suffix {
 		cursor: pointer;
 	}
+
+	.woocommerce-image-gallery, .components-form-file-upload {
+		display: inline-block;
+		vertical-align: middle;
+		p {
+			margin-bottom: $gap-smallest;
+		}
+		button {
+			color: $gutenberg-blue;
```

Comment:
This doesn't exist, I noticed you created this before in another css file, so maybe do something similar?
`$gutenberg-blue: var(--wp-admin-theme-color);`?

Reply:
Good call @louwie17! I fixed it in the commit [475cc59](https://github.com/woocommerce/woocommerce/pull/40835/commits/475cc59847837dd63a8cc963c26811c513962b0e)